app/src/main/java/org/systers/mentorship/view/adapters/RelationPagerAdapter.kt

```diff
@@ -12,47 +15,35 @@ import org.systers.mentorship.view.fragments.TasksFragment
 
 @SuppressLint("ValidFragment")
 /**
- * This is the [FragmentPagerAdapter] responsible for the configuration each fragment assigned to
+ * This is the [FragmentStateAdapter] responsible for the configuration each fragment assigned to
  * each tabs. One tab displays the details of the current mentorship relation and the other provides
  * a detailed information about the tasks.
- * @param fm fragment manager
  */
-class RelationPagerAdapter(fm: FragmentManager, private var mentorshipRelation: Relationship) : FragmentPagerAdapter(fm) {
+class RelationPagerAdapter(
+    fragmentManager: FragmentManager,
+    lifecycle: Lifecycle,
+    private val mentorshipRelation: Relationship
+) : FragmentStateAdapter(fragmentManager, lifecycle) {
 
     /**
      * This class represents the number and index of each tab of the layout
      */
     enum class TabsIndex(val value: Int) {
-        DETAILS(0),
-        TASKS(1)
+        DETAILS(0), TASKS(1)
```

Comment:
I think you reformatted the entire file. Since this is a PR pertaining to moving to ViewPager2, these changes will have to be removed.

Reply:
That's right, I'll fix it.